src/kontrol/__main__.py

```diff
@@ -790,6 +792,13 @@ def parse(s: str) -> list[T]:
         action='store_true',
         help='Reinitialize CFGs even if they already exist.',
     )
+    prove_args.add_argument(
+        '--setup-version',
+        dest='setup_version',
+        default=None,
+        type=int,
+        help='Instead of reinitializing the test setup with together the test proof, select the setup version to be reused during the proof.',
```

Comment:
```suggestion
        help='Instead of reinitializing the test setup together with the test proof, select the setup version to be reused during the proof.',
```

Reply:
Or maybe it's also worth highlighting that it should be used together with `--reinit`, like we say
```
  --break-on-basic-blocks
                        Store a node for every EVM basic block (implies --break-on-calls).
```
E.g., something like
```
"When reinitializing the proof with '--reinit', do not reinitialize 'setUp' and use the specified version instead"
```
Or you can just add `(implies --reinit)`, or something like that, but it looks good as it is too.